.github/workflows/build-images.yml

```
@@ -53,6 +52,9 @@ on:
       minimal-vertica-image:
         description: "The image name of the vertica server, but with optional software removed"
         value: ${{ jobs.build-server-minimal.outputs.image }}
+      v2_vertica_image:
```

I think this needs to be dashes not underscores. It has to match the variable name that you reference in e2e.yml
```suggestion
      v2-vertica-image:
```